File: app/src/main/java/com/example/rlapp/apimodel/Module/ModuleResponse.java
package com.example.rlapp.apimodel.Module;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ModuleResponse {

    @SerializedName("success")
    @Expose
    private Boolean success;
    @SerializedName("statusCode")
    @Expose
    private Integer statusCode;
    @SerializedName("data")
    @Expose
    private List<ModuleList> data;

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public Integer getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(Integer statusCode) {
        this.statusCode = statusCode;
    }

    public List<ModuleList> getData() {
        return data;
    }

    public void setData(List<ModuleList> data) {
        this.data = data;
    }

}

File: app/src/main/java/com/jetsynthesys/rightlife/ai_package/model/ModuleResponse.kt
package com.jetsynthesys.rightlife.ai_package.model

import com.google.gson.annotations.SerializedName

data class ModuleResponse(
    @SerializedName("success"    )
    var success    : Boolean? ,
    @SerializedName("statusCode" )
    var statusCode : Int?,
    @SerializedName("data"       )
    var data       : ArrayList<ModuleData> = arrayListOf()
)
data class ModuleData(
    @SerializedName("_id"       )
    var _id       : String?,
    @SerializedName("title"         )
    var title         : String?,
    @SerializedName("desc"       )
    var desc       : String?,
    @SerializedName("image"        )
    var image        : String?,
    @SerializedName("isSelectedModule" )
    var isSelectedModule : Boolean?
)
